content/2021-12-14-brigade-2.1.0.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Brigade 2.1.0"
+slug: "2021-12-14-brigade-2.1.0"
+description: "Brigade 2.1.0 has been released"
+date: "2021-12-14"
+author: "Kent Rancourt"
+summary: "Brigade 2.1.0 has been released."
+---
+
+Good day, Brigadiers! Today we have _minor_ news for you -- that is we're announcing our first minor release of Brigade 2.
+
+We certainly never imagined Brigade v2.1.0 being released only two weeks after Brigade v2.0.0 -- and we always imagined v2.1.0 being quite feature-packed. Instead, this release is quite _boring_, but we love keeping our community updated, so here we are!
```

Comment:
I would change "boring" to "minimal" since boring implies a value judgement that's subjective and can be read as more negative than we'd want here.

Reply:
Changed it to "Instead, this release contains only one small feature..."

wdyt?

Reply:
I appreciate the change. Despite aiming at oneself, negativity can dim the light of any potential contributor. We want them excited!

Reply:
Makes good sense. Thank you, @bridgetkromhout !